commands/summary.go

```diff
@@ -127,32 +110,22 @@ func addServerData(zipW *zip.Writer, serverURL *url.URL, serverInsecureTLS bool)
 	return nil
 }
 
-func addClientFile(zipW *zip.Writer, name string) {
+func addFileToZip(zipW *zip.Writer, fpath, name string) {
 	if name == "" {
 		return
 	}
 
 	b, err := ioutil.ReadFile(name) //nolint:gosec
 	if err != nil {
-		logrus.Debugf("%s", err)
+		logrus.Warnf("%s", err)
```

Comment:
It is expected (and handled) that not all files are present for all distribution methods (Docker, AMI, OVF).

Reply:
Which files might be not present? I see that we call this method only for client config and pprof files.